### PR TITLE
Add release notes for 2.10.9

### PR DESCRIPTION
**Pull request by madkinsz**
[Rendered](https://github.com/PrefectHQ/prefect/blob/release-notes-2.10.9/RELEASE-NOTES.md#release-2109)

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,5 +1,42 @@
...
+## Release 2.10.9
+
+### Worker logs can now be seen on the flow run page
+
+Workers now link relevant logs to specific flow runs, allowing you to view infrastructure related logs on your flow run page.
+
+<img width="1294" alt="Process worker logs" src="https://github.com/PrefectHQ/prefect/assets/2586601/658c2883-69f7-4ee0-abf6-a20ee4723b3a">
+
+
+You'll see generic logs from all worker types and following this release the integration worker implementations e.g. for Kubernetes will be updated to send additional rich logs to give you insight into the behavior of flow run infrastructure.
```
Comment (serinamarie) [inline RELEASE-NOTES.md:12]:
```suggestion
You'll see generic logs from all worker types. Integration worker implementations such as Kubernetes workers will be updated to send additional rich logs to give you insight into the behavior of flow run infrastructure.
```

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -1,5 +1,42 @@
...
+- Flush logs before exiting deployed flow run processes — https://github.com/PrefectHQ/prefect/pull/9516
+- Update worker to be able to include itself as a related resource — https://github.com/PrefectHQ/prefect/pull/9531
+
+### Fixes
+- Fix bug where `SIGTERM` was not properly captured as a flow run crash — https://github.com/PrefectHQ/prefect/pull/9498
+- Fix pass of optional parameters to API in `client.create_work_queue` — https://github.com/PrefectHQ/prefect/pull/9521
+
+### Documentation
+- Add tip about flow run level concurrency — https://github.com/PrefectHQ/prefect/pull/9490
+- Add `on_failure` example to docs — https://github.com/PrefectHQ/prefect/pull/9511
```
Comment (serinamarie) [inline RELEASE-NOTES.md:30]:
```suggestion
- Add documentation on `on_failure` flow run state change hook — https://github.com/PrefectHQ/prefect/pull/9511
```

**Review by serinamarie — COMMENTED** [review]
✨

File: RELEASE-NOTES.md [hunk 3]
```diff
@@ -1,5 +1,42 @@
...
+<img width="1294" alt="Process worker logs" src="https://github.com/PrefectHQ/prefect/assets/2586601/658c2883-69f7-4ee0-abf6-a20ee4723b3a">
+
+
+You'll see generic logs from all worker types. Integration worker implementations such as Kubernetes workers will be updated to send additional rich logs to give you insight into the behavior of flow run infrastructure.
+
+See https://github.com/PrefectHQ/prefect/pull/9496 for details.
+
+### Enhancements
+- Handle `SIGTERM` received by agent gracefully — https://github.com/PrefectHQ/prefect/pull/8691
+- Global flow and task retries and retry delay seconds — https://github.com/PrefectHQ/prefect/pull/9171
```
Comment (serinamarie) [inline RELEASE-NOTES.md:18]:
```suggestion
- Add settings for global flow and task retries and retry delay seconds — https://github.com/PrefectHQ/prefect/pull/9171
```

File: RELEASE-NOTES.md [hunk 4]
```diff
@@ -1,5 +1,42 @@
...
+- Add `include_submodules` option to `git_clone_project` step — https://github.com/PrefectHQ/prefect/pull/9504
```
Comment (serinamarie) [inline RELEASE-NOTES.md:19]:
```suggestion
- Add support for Github Submodules in Prefect projects — https://github.com/PrefectHQ/prefect/pull/9504
```

File: RELEASE-NOTES.md [hunk 5]
```diff
@@ -1,5 +1,42 @@
...
+- Add wrapper for exceptions encountered while resolving parameter inputs — https://github.com/PrefectHQ/prefect/pull/8584
+- Flush logs before exiting deployed flow run processes — https://github.com/PrefectHQ/prefect/pull/9516
```
Comment (serinamarie) [inline RELEASE-NOTES.md:21]:
```suggestion
- Flush logs before exiting deployed flow run processes for useful failing debugging  — https://github.com/PrefectHQ/prefect/pull/9516
```

File: RELEASE-NOTES.md (hunk 4 above)
Comment (madkinsz) [reply RELEASE-NOTES.md:19]: I prefer the previous in this case. Or:

```suggestion
- Add `include_submodules` option to `git_clone_project` step for submodule support — https://github.com/PrefectHQ/prefect/pull/9504
```

but I don't think we really need it

Comment (madkinsz) [reply RELEASE-NOTES.md:19]: I prefer the previous in this case. Or:

```suggestion
- Add `include_submodules` option to `git_clone_project` step for submodule support — https://github.com/PrefectHQ/prefect/pull/9504
```

but I don't think we really need it

Comment (serinamarie) [reply RELEASE-NOTES.md:19]: Makes sense

**Comment by madkinsz** [comment]
Thanks for the nudge on those! I did not use your suggestions but I did rewrite those three items hopefully they're better.

**Review by serinamarie — APPROVED** [review]
LGTM :)

**Pull request merged**